### PR TITLE
Fix scale and altitude features

### PR DESCRIPTION
**Pull request by gchoqueux**
## Description
fix(Feature): wrong altitude and altitude limits.
fix(Feature2Mesh): set scale transformation from `FeatureCollection`.

### CODE REVIEW

File: src/Converter/Feature2Mesh.js [hunk 1]
```diff
@@ -484,6 +484,8 @@ export default {
 
             group.quaternion.copy(collection.quaternion);
             group.position.copy(collection.position);
+            group.scale.copy(collection.scale);
+            group.feature = collection;
```
Comment (gmaillet) [inline src/Converter/Feature2Mesh.js]: why the collection is referenced in group.feature?

File: src/Core/Feature.js [hunk 2]
```diff
@@ -157,6 +157,11 @@ export class FeatureGeometry {
      * @param {THREE.Vector3} [normal=THREE.Vector3(0,0,1)] the normal on coordinates.
      */
     pushCoordinatesValues(feature, long, lat, alt = 0, normal = defaultNormal) {
+        if (this.size == 3) {
+            // set altitude from context
+            const base_altitude = feature.style[typeToStyleProperty[feature.type]].base_altitude;
+            alt = isNaN(base_altitude) ? base_altitude(this.properties) : base_altitude;
+        }
```
Comment (gmaillet) [inline src/Core/Feature.js]: is it possible to factorize this with pushCoordinates?

File: src/Converter/Feature2Mesh.js (hunk 1 above)
Comment (gchoqueux) [reply src/Converter/Feature2Mesh.js]: removed

File: src/Core/Feature.js (hunk 2 above)
Comment (gchoqueux) [reply src/Core/Feature.js]: refactorized

**Review by ign-packo — APPROVED** [review]
test with a FeatureGeometryLayer based on VectorTilesSource: scale and altitude.min/max are actually updated

**Review by gmaillet — APPROVED** [review]
test with a FeatureGeometryLayer based on VectorTilesSource: scale and altitude.min/max are actually updated

**Pull request merged**